### PR TITLE
Add trailing slash to remove redirects.

### PR DESCRIPTION
**Pull request by tobinjt**
When you request /tags/foo, the web server will redirect you to
/tags/foo/ so the browser needs to make two requests.  Add the trailing
slash in the link so that only one request is needed.

### CODE REVIEW

**Comment by crachel** [comment]
Also needs it for /categories/foo/ since both point to directories.

**Comment by tobinjt** [comment]
> Also needs it for /categories/foo/ since both point to directories.

Good point, added.  Thanks :)

**Comment by luizdepra** [comment]
Hmm... I need to check if there is a better way to solve this problem ratter than concatenating a `/`.

File: layouts/partials/taxonomy/categories.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
     {{- if gt $index 0 }}
       <span class="separator">•</span>
     {{- end }}
-    <a href="{{ ( print "categories/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
+    <a href="{{ ( print "categories/" ( . | urlize ) "/" ) | relLangURL }}">{{ . }}</a>
```
Comment (luizdepra) [inline layouts/partials/taxonomy/categories.html]:
```suggestion
    <a href="{{ ( printf "categories/%s/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
```
It is better this way.

File: layouts/partials/taxonomy/tags.html [hunk 2]
```diff
@@ -4,6 +4,6 @@
     {{- if gt $index 0 }}
       <span class="separator">•</span>
     {{- end }}
-    <a href="{{ ( print "tags/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
+    <a href="{{ ( print "tags/" ( . | urlize ) "/" ) | relLangURL }}">{{ . }}</a>
```
Comment (luizdepra) [inline layouts/partials/taxonomy/tags.html]:
```suggestion
    <a href="{{ ( printf "tags/%s/" ( . | urlize ) ) | relLangURL }}">{{ . }}</a>
```
Same here.

**Comment by luizdepra** [comment]
@tobinjt, I just suggested two fixes.
Also, add yourself into `CONTRIBUTORS.md`.

**Comment by tobinjt** [comment]
I've accepted those two fixes and added myself to CONTRIBUTORS.md, thanks!

**Comment by luizdepra** [comment]
Thanks. :)

**Pull request merged**